cfgen/cfgen

```diff
@@ -2082,7 +2110,18 @@ def build_cluster(cluster_desc: Dict[str, Any]) -> Cluster:
         pool_t = pool_type(pool)
         if pool_t == PoolT.sns:
             # For SNS pools create the aux pool list
-            aux_pool_list(pool, conf)
+            disks: List[DiskRef] = []
+            disk_refs = get_pool_diskrefs(pool)
```

Comment:
disk_refs is used in `if disk_refs is None` only. Is that correct?

So if `disk_refs` is not None, then the value is ignored.

Reply:
Yes, correct. We need to generate disk_refs only if it is not present in pool description